docs/docs/smart-contracts/PSP22/Utils/token-timelock.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 1
+title: PSP22 Token Timelock
+---
+
+This example shows how you can reuse the implementation of [PSP22 Token Timelock](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp22/utils/token_timelock) utility for [PSP22](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp22).
```

Comment:
Could you describe please how to use that contract?=) Because it is not clear for the user how to user that=)